### PR TITLE
[v8] Fix copyright headers

### PR DESCRIPTION
**Pull request by JulianGro**

### CODE REVIEW

File: tests/script-engine/src/ScriptEngineTests.h [hunk 1]
```diff
@@ -1,12 +1,9 @@
 //
-//  AudioTests.h
-//  tests/audio/src
-//
-//  Created by Dale Glass on 27/8/2022
```
Comment (ksuprynowicz) [inline tests/script-engine/src/ScriptEngineTests.h:5]: Can we keep this part?

**Review by ksuprynowicz — CHANGES_REQUESTED** [review]
Can we keep "Created by Dale Glass on 27/8/2022" on AudioTests.h? I see no reason to remove it.

File: tests/script-engine/src/ScriptEngineTests.h (hunk 1 above)
Comment (JulianGro) [reply tests/script-engine/src/ScriptEngineTests.h:5]: I just didn't know if it was created by Dale because that was clearly just copied from AudioTests.h

Comment (ksuprynowicz) [reply tests/script-engine/src/ScriptEngineTests.h:5]: It was indeed created by Dale

**Comment by JulianGro** [comment]
This is already in the v8 branch, because my git client apparently pushes to the wrong remote sometimes..

**Pull request closed without merging**